src/highlight-vue.ts

```diff
@@ -0,0 +1,78 @@
+import { ref, h, computed, defineComponent, Plugin } from 'vue'
+import hljs from 'highlight.js/lib/core'
+
+function escapeHtml(value: string): string {
```

Comment:
Curious, what was the issue with this utility living in `lib/utils`?

Reply:
No real reason, just did it on a whim. Want me to put it back?

Reply:
I also just noticed vue provides an `escapeHtml` function as well https://github.com/vuejs/vue-next/blob/master/packages/shared/src/escapeHtml.ts

Reply:
Since it escapes all 5 I think we could use that if it lightens the payload a bit.

When is Vue 3 going to be released?

Reply:
It was released Sept last year

Reply:
Ah, they just plan to keep their `next` tagging in NPM forever then I suppose? :-)

Reply:
Pretty much yea